codesign/lib/verify.dart

```diff
@@ -0,0 +1,185 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:logging/logging.dart';
+import 'package:process/process.dart';
+
+const String kNotNotarizedMessage = 'test-requirement: code failed to satisfy specified code requirement(s)';
+
+enum VerificationResult {
+  unsigned,
+  codesignedOnly,
+  codesignedAndNotarized,
+}
+
+class VerificationService {
+  VerificationService({
```

Comment:
Should this also check that you're on a Mac?

Reply:
done